use.md

```diff
@@ -4,13 +4,17 @@ import TagTile from ".vitepress/components/TagTile.vue";
 
 # Use
 
-> We use free software wherever possible!  
-
 The free software ecosystem offers countless solutions and applications for modern software development.
 Without the use of free software, modern software development is no longer imaginable.
 You can find free software in almost all modern software products.
 The importance and influence of free software on software development is therefore indispensable and undeniable.
 
+Our [strategic guidelines](principles#legal-and-political-requirements) also provide for this:
+
+> If economically and technologically or strategically sensible, LHM prioritizes the use of open source solutions, in particular to avoid company dependencies.
```

Comment:
```suggestion
> If economically and technologically or strategically sensible, the city of munich prioritizes the use of open source solutions, in particular to avoid company dependencies.
```

Reply:
your version sounds better, but I would stick to the direct translation of our [strategic guidelines](https://opensource.muenchen.de/principles.html#legal-and-political-requirements)